resources/views/vendor/notifications/email.blade.php

```diff
@@ -45,7 +45,7 @@
 @if (! empty($salutation))
 {{ $salutation }}
 @else
-Regards,<br>{{ config('app.name') }}
+Regards,<br> Rose Vouchers
```

Comment:
Will this need to be configurable?

Reply:
Yes, but i need confirmation on what it should be as an aggregate *thing* i've gone for "rose vouchers" for now.

Reply:
Will add the config version back in and change the app.name in .env